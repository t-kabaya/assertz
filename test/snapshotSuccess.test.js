const { snap } = require('../index')

snap(777, 'number snap')
