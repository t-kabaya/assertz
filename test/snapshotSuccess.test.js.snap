{"object snap":"{\"foo\":\"foo\"}","number snap":"777"}